.gitignore

```diff
@@ -11,3 +11,7 @@ scTopics
 functions.csv
 artifacts/
 allFiredEvents
+
+# geth and parity specific files
+password
```

Comment:
Aren't these files needed to run?

Reply:
No, not needed I was just using that file to store passwords for accounts for geth/parity. I just didn't want to be committing any password files.